core/include/masks/mask_identifier.hpp

```diff
@@ -0,0 +1,22 @@
+/** Detray library, part of the ACTS project (R&D line)
+ * 
+ * (c) 2020 CERN for the benefit of the ACTS project
+ * 
+ * Mozilla Public License Version 2.0
+ */
+#pragma once
+
+namespace detray {
+
+	/** Enumerate different mask types for convenience
+	**/
+	enum mask_identifier : unsigned int {
+		e_rectangle2 = 0,
+		e_trapezoid2 = 1,
+		e_ring2 = 2,
+		e_cylinder3 = 3,
+		e_single3 = 4,
+		e_annulus = 5,
```

Comment:
Probably we should name this `annulus2` to indicate in the name that it acts on a local position as with `rectangle2` etc.